scripts/sdf-glyph-foundry/0.1.0/.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: cpp
```

Comment:
`language: generic` works here and should allow the machines to boot a bit quicker. This is the reason I default to it instead of `cpp` which adds nothing for our usage.